docs/_config.yml

```diff
@@ -0,0 +1,5 @@
+title: Language Support for Apache Camel by Red Hat
+logo: https://raw.githubusercontent.com/camel-tooling/camel-lsp-client-vscode/main/icons/icon128.png
+description: This is a client implementation of the Apache Camel Language Server Protocol for Visual Studio Code
```

Comment:
@apupier is this interpretation legally right? `....of the Apache Camel Language Server Protocol for Visual Studio Code`

Reply:
maybe more safe variant should be something like 
`This is the client implementation of the Language Server for Apache Camel for Visual Studio Code` ??